deploy/roles/shared/handlers/main.yml

```diff
@@ -40,6 +40,7 @@
 # rhel7
 - name: reload-daemon
   systemd:
+    name: "{{ pp_servicename }}"
```

Comment:
@rioh, see my comment on the other PR, I didn't think this was necessary, but let me know if you determined otherwise!

Reply:
I think that option is only available with Ansible 2.4.

Reply:
Ah, interesting.  OK, leave it as is for now, I will merge, thanks!